### ISSUE TITLE
[Bug] <Older comics are losing their categories>

### ISSUE THREAD

**Issue by MatanBuljo** [inorichi/tachiyomi#3671]
## Device information
* Tachiyomi version: App version: 0.10.4 (standard, 65b32ddeb, 50)
Android version: 9 (SDK 28)
Android build ID: PKQ1.180904.001
Device manufacturer: Xiaomi
Device model: Redmi Note 7

## Steps to reproduce
I'm actually not sure if I did anything, I was doing everything normally.

### Expected behavior
The comics stay in their categories, where I put them.

### Actual behavior
Comics are being decategorised regularly.

## Other details
Basically, I love to have order in my comics. So I have a category for finished, unfinished and a category for every main character. There are currently 146 categories in my app. Everything was ok until a few days ago, when I noticed some unfinished comics, not in the unfinished category. Also, some comics that I finished and assigned to their respective character categories months ago, aren't in their respective character categories any more.

**Comment by arkon** [comment]
Unactionable without consistent reproduction steps.

**Comment by arkon** [comment]
No response, closing.
